publisher-sdk-tests/src/androidTest/java/com/criteo/publisher/integration/DfpHeaderBiddingFunctionalTest.java

```diff
@@ -597,9 +599,10 @@ public void onAdLoaded() {
 
       @Override
       public void onAdFailedToLoad(int i) {
+        Log.w(TAG, "onAdFailedToLoad, for reason: " + i +
```

Comment:
Nit: you can also use the Logger class (see https://github.com/criteo/android-publisher-sdk/blob/7c1b0b91f8a9dc78ba25b8850d51d5d51d244516/publisher-sdk/src/main/java/com/criteo/publisher/bid/LoggingBidLifecycleListener.java#L32) that handle %s, %d, ... formatting with varargs.

Reply:
Done